modules/ssh/ssh.go

```diff
@@ -169,12 +169,12 @@ func Listen(host string, port int) {
 		filePath := filepath.Dir(keyPath)
 
 		if err := os.MkdirAll(filePath, os.ModePerm); err != nil {
-			log.Error(4, "Fail to create dir %s: %v", filePath, err)
+			log.Error(4, "Failed to create dir %s: %v", filePath, err)
 		}
 
 		_, stderr, err := com.ExecCmd("ssh-keygen", "-f", keyPath, "-t", "rsa", "-N", "")
 		if err != nil {
-			panic(fmt.Sprintf("Fail to generate private key: %v - %s", err, stderr))
+			panic(fmt.Sprintf("Failed to generate private key: %v - %s", err, stderr))
```

Comment:
Good catch. I didn't even notice the failure message was wrapped in a panic call. I removed two other uses of `panic()` in the latest commit.